src/fsharp/vs/ServiceUntypedParse.fs

```diff
@@ -340,8 +340,13 @@ type FSharpParseFileResults(errors : FSharpErrorInfo[], input : Ast.ParsedInput
  
         ErrorScope.Protect 
             Range.range0 
-            (fun () -> findBreakPoints() |> List.tryLast)
-            (fun _msg -> None)   
+            (fun () -> 
+                let locations = findBreakPoints()
+                
+                match locations |> List.filter (fun m -> rangeContainsPos m pos) with
+                | [] -> Seq.tryHead locations
```

Comment:
Okay when you send the cleanup, can you add a comment describing this.


Thanks.